applications/FemToDemApplication/python_scripts/FemDemStaticSolver.py

```diff
@@ -8,7 +8,7 @@
 KratosMultiphysics.CheckForPreviousImport()

```

Comment:
```suggestion
```

btw this is no longer used and will probably be removed soon

Reply:
mm I'll check and remove in the future then